psalm-baseline.xml

```diff
@@ -1,5 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <files psalm-version="4.10.0@916b098b008f6de4543892b1e0651c1c3b92cbfa">
+  <file src="polyfill/test/Environment.php">
+    <ParseError occurrences="1">
+      <code>Environment</code>
+    </ParseError>
+  </file>
+  <file src="polyfill/test/Flags.php">
+    <ParseError occurrences="2">
+      <code>Flags</code>
+      <code>case</code>
+    </ParseError>
+  </file>
+  <file src="polyfill/test/Orientation.php">
+    <ParseError occurrences="2">
+      <code>Orientation</code>
+      <code>case</code>
+    </ParseError>
+  </file>
```

Comment:
For other 8.1 features i used eval to generate classes while runtime, see https://github.com/laminas/laminas-code/blob/4.5.x/test/Generator/PropertyGeneratorTest.php#L405

Reply:
Yeah, these will need to be moved to strings, for now: reason being that psalm chokes in a clean environment too.

Reply:
5e6e3d013c32b3681533143b8dc7bc27fcd25d1d